packages/css/src/atoms/selects/select.scss

```diff
@@ -18,7 +16,7 @@
     }
 
     &__selected {
-        width: 100%;
+        width: Min(100%, tokens.$width);
```

Comment:
Please, follow the convention of CSS to favor lowercase, even in the `min()` function.

Reply:
Resolved with https://github.com/bod-endava/e-toast-ds/pull/54/commits/b4c973acf364b3e2704dca279d0eb848e8ce8073